src/lib.rs

```diff
@@ -198,34 +196,20 @@ pub use crate::scalar::Scalar;
 /// Trait describing something that promises to be a 32-byte random number; in particular,
 /// it has negligible probability of being zero or overflowing the group order. Such objects
 /// may be converted to `Message`s without any error paths.
+#[deprecated(since = "0.29.0", note = "this will be removed soon")]
```

Comment:
In 00b66df56d0383ae4090b98cdd515660a98ab1f1:

Not sure if we can easily put a multi-line message here, but let's suggest that users directly implement `From<T> for Message` for their type if their goal is to have easy convertibility to messages, and explain that `sign_ecdsa` and `sign_schnorr` can now directly take such types rather than requiring a manual conversion to `Message`.

Reply:
Oh, ok, we can drop the "no need for manual conversion" since we still need a manual conversion for now. We should suggest people write `Message::from` though.  Later clippy will tell them that they can drop the `Message::from`

If they just do `.into()` this will work today but break when we change our signing functions to take generics, because Rust won't know the target type to convert to.